microhapulator/data/template.html

```diff
@@ -25,14 +30,34 @@
             table {
                 margin: 12pt auto;
                 width: 100%;
+                display: block;
+                max-width: -moz-fit-content;
+                max-width: fit-content;
+                overflow-x: auto;
             }
             table.half {
                 margin: 12pt auto 12pt 0;
                 width: 50%;
             }
             td, th {
-                padding: auto 5px;
+                padding-right: 10px;
+                white-space: nowrap;
             }
+            ::-webkit-scrollbar {
+            width: 5px;
+            height: 5px;
+        }
+
+        ::-webkit-scrollbar-thumb {
+            background: gray;
+            border-radius: 10px;
+        }
+        ::-webkit-scrollbar-track{
+           -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
+	        border-radius: 10px;
+	        background-color: #F5F5F5;
+        }
```

Comment:
This works well in Safari on my Mac, but at a minimum we'll also want to try FireFox on Mac and Windows, and possibly (though lower priority) Chrome and Edge.

Reply:
I tested on FireFox (Mac and Windows) and Edge and it works. The only issue is that the scroll bars in FireFox on Mac aren't visible until you start trying to scroll. This can be changed by going to apple menu -> System Preferences -> General,  and setting 'Show scroll bars' to Always. However, I assume most people probably have the default option of 'When scrolling'.

Reply:
Sounds good!